internal/controller/nats/deprovisioner.go

```diff
@@ -116,8 +117,10 @@ func (r *Reconciler) createAndConnectNatsClient(nats *nmapiv1alpha1.NATS) error
 }
 
 func (r *Reconciler) deletePVCsAndRemoveFinalizer(ctx context.Context,
-	nats *nmapiv1alpha1.NATS, log *zap.SugaredLogger) (kcontrollerruntime.Result, error) {
+	nats *nmapiv1alpha1.NATS, log *zap.SugaredLogger,
+) (kcontrollerruntime.Result, error) {
 	labelValue := nats.Name
+	//nolint: godox
```

Comment:
Could this be solved by contacting the person who added the original `ToDo`? I think we also talked in the team, that we do not want to have `ToDo` messages left in the code. Maybe this can be converted to a ticket.

Reply:
Sure

Reply:
Could you take care of this?

Reply:
I am on it. @muralov is already checking it.

Reply:
@muralov's answer: I’ve checked that we use the “eventing” label to delete PVCs, and therefore we should not delete it right now.

Reply:
we should agree on a way we handle ToDos in code